src/App.tsx

```diff
@@ -3,14 +3,25 @@ import './App.css';
 import { useState } from 'react';
 import { UserData } from './Interfaces';
 import LoginForm from './Login';
-import MenuBar from './components/layout/MenuBar';
+// import { getUser } from './apiCalls';
+import MenuBar from './components/layout/MenuBar'
+// import { BrowserRouter } from 'react-router-dom';
```

Comment:
Is there a reason we are commenting out lines 6 & 8 and not outright deleting them? I assume it's because we don't want to declare something and not use it to avoid errors, but if this assumption is incorrect we should delete before production.

Reply:
It makes sense to comment out these lines. Although deleting them as a whole would be better.They're not being used in the file, and typescript won't allow these in if they're not being used.